AndelaHacks/See-the-Light
# #20190325 IP / domain reputation checker

**What does this PR do?**
Adds a function to check the IP/Domain of news items for reputation

**Description of Tasks to be completed**
Create and test the check reputation function
Integrate the function to check and return reputation score

**How should this be manually tested?**
- Clone this repo by running git clone `https://github.com/AndelaHacks/See-the-Light.git`
- Activate virtual environment and install requirements
- Run the flask app
- Consume the api using postman

**Branch Name**
ft-domain-checker-20190325
